ApiBundle/Transformer/NodeGroupRoleTransformer.php

```diff
@@ -52,25 +66,43 @@ public function transform($nodeGroupRole)
     }
 
     /**
-     * @param FacadeInterface             $facade
-     * @param NodeGroupRoleInterface|null $source
+     * @param FacadeInterface     $nodeRoleFacade
+     * @param GroupInterface|null $group
      *
      * @throws RoleNotFoundHttpException
-     * @return mixed
+     * @throws TransformerParameterTypeException
+     *
+     * @return null|NodeGroupRoleInterface
      */
-    public function reverseTransform(FacadeInterface $facade, $source = null)
+    public function reverseTransform(FacadeInterface $nodeRoleFacade, $group = null)
```

Comment:
I would modify the signature by adding a parameter : ($nodeRoleFacade, $source = null, $group = null).

This way you will modify the source which is given by the caller of the method and you won't have to find it inside this method